compiler/rustc_trait_selection/src/lib.rs

```diff
@@ -22,6 +22,8 @@
 #![feature(never_type)]
 #![feature(type_alias_impl_trait)]
 #![recursion_limit = "512"] // For rustdoc
+// #![deny(rustc::untranslatable_diagnostic)]
+// #![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
If we're going to just land part of this crate's migration, then could we remove these comments?

Reply:
Done :)